content/v2.0/reference/flux/stdlib/experimental/aligntime.md

```diff
@@ -0,0 +1,88 @@
+---
+title: experimental.alignTime() function
+description: >
+  The `experimental.alignTime()` function aligns input tables to a common start time.
+menu:
+  v2_0_ref:
+    name: experimental.alignTime
+    parent: Experimental
+weight: 302
+---
+
+The `experimental.alignTime()` function aligns input tables to a common start time.
+
+
+_**Function type:** Transformation_
+
+```js
+import "experimental"
+
+experimental.alignTime(
+  alignTo: 1970-01-01T00:00:00.000000000Z
+)
+```
+
+## Parameters
+
+### alignTo
+The **UTC time** to align tables to.
+Default is `1970-01-01T00:00:00Z`.
+
+_**Data type:** Time_
+
+## Examples
+
+### Compare values month-over-month
+```js
+import "experimental"
+
+from(bucket: "example-bucket")
+  |> range(start: -12mo)
+  |> filter(fn: (r) => r._measurement == "example-measurement")
+  |> window(every: 1mo)
+  |> experimental.alignTime()
+```
+
+**Given the following input:**
+
+| _time                | _value |
+|:-----                | ------:|
+| 2020-01-01T00:00:00Z | 32.1   |
+| 2020-01-02T00:00:00Z | 32.9   |
+| 2020-01-03T00:00:00Z | 33.2   |
+| 2020-01-04T00:00:00Z | 34.0   |
+| 2020-02-01T00:00:00Z | 38.3   |
+| 2020-02-02T00:00:00Z | 38.4   |
+| 2020-02-03T00:00:00Z | 37.8   |
+| 2020-02-04T00:00:00Z | 37.5   |
+
+**The following functions output:**
+
+```js
+//...
+  |> window(every: 1mo)
+  |> alignTime(alignTo: 2020-01-01T00:00:00Z)
+```
+
+{{< flex >}}
+{{% flex-content %}}
+| _time                | _value |
+|:-----                | ------:|
+| 2020-01-01T00:00:00Z | 32.1   |
+| 2020-01-02T00:00:00Z | 32.9   |
+| 2020-01-03T00:00:00Z | 33.2   |
+| 2020-01-04T00:00:00Z | 34.0   |
+{{% /flex-content %}}
+{{% flex-content %}}
+| _time                | _value |
+|:-----                | ------:|
+| 2020-01-01T00:00:00Z | 38.3   |
+| 2020-01-02T00:00:00Z | 38.4   |
+| 2020-01-03T00:00:00Z | 37.8   |
+| 2020-01-04T00:00:00Z | 37.5   |
```

Comment:
2020-02...?

Reply:
No, because they're all aligned to the same start time now.

Reply:
No, if there's one table, the time in that will table will be updated to start at whatever `alignTo` is set to, but all subsequent points' time values are updated relative to the new start time of the table.

Here's a screenshot of what this function does. Both graphs represent the same data, just the 2nd one aligns all the tables by time so you can visually compare them.

![image](https://user-images.githubusercontent.com/181484/78398450-56d95900-75b0-11ea-88c2-b8dc4f3e7954.png)